### ISSUE TITLE
训练突然停止，可以接着以前的训练么？

### ISSUE THREAD

**Issue by hexin6688** [qqwweee/keras-yolo3#658]
Epoch 9/50
252/252 [==============================] - 386s 2s/step - loss: 346.2141 - val_loss: nan
Epoch 10/50
251/252 [============================>.] - ETA: 1s - loss: 286.6874
Process finished with exit code -1
训练时，突然就中止了，难道只能重新训练么？还有，为什么会突然中止呢？
